lib/msf/core/payload/php.rb

```diff
@@ -82,7 +82,7 @@ def php_system_block(options = {})
     passthru = "
       if(#{is_callable}('passthru')&&!#{in_array}('passthru',#{dis})){
         ob_start();
-        passthru(#{cmd});
+        `#{cmd}`;
```

Comment:
Could we get some clarification/reasoning behind this change?

This now relies on `shell_exec`. However, if we get to this point, we know that `shell_exec` is disabled, based on the lines above: https://github.com/rapid7/metasploit-framework/pull/19445/files#diff-964542ddc7e6e035ff04c32c78aae505d165933654eee96f05002f955d538d65R78-R81

Reply:
The reason is me misremembering what ``` was synonym of, sorry :/